tasks/Makefile.projects

```diff
@@ -0,0 +1,16 @@
+## Add a tool project
+projects/add/tool/%:
+	git submodule add git@github.com:cloudposse/$(ARG3).git projects/tools/$(ARG3)
+
+## Add a terraform project
+projects/add/terraform/%:
+	git submodule add git@github.com:cloudposse/$(ARG3).git projects/$(ARG2)/$(ARG3)
```

Comment:
this is probably the same 

```
projects/$(ARG2)/$(ARG3)
projects/terraform/$(ARG3)
```

but you use a few of them with the second part hardcoded
```
projects/tools/$(ARG3)
```